test/iron-dropdown-scroll-manager.html

```diff
@@ -47,14 +41,14 @@
 
       setup(function() {
         parent = fixture('DOMSubtree');
-        childOne = parent.querySelector('#ChildOne');
-        childTwo = parent.querySelector('#ChildTwo');
-        grandChildOne = parent.querySelector('#GrandchildOne');
-        grandChildTwo = parent.querySelector('#GrandchildTwo');
+        childOne = Polymer.dom(parent.root).querySelector('#ChildOne');
```

Comment:
i think this is just `parent.$$.ChildOne`, which is less verbose, maybe? (here and below)